### DISCUSSION TITLE
Feedback for published site redesign

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#514] [2. Published content]
### Ask a question, start a conversation, or make an announcement

While the new design is mostly fine (It's especially nice that the PDF export isn't buried in a menu anymore), there are a few display issues.

**Table issues**
I've been saying this for a long time, and reported it before, but table cell content really needs to be vertically aligned to the top of the cell, not the middle of the cell. I thought when I previewed the new design it was vertically aligned to the top, but now that it's out it's center aligned again. It makes the table content very difficult to read.

There is not enough space above the tables.

Please restore the shading to the heading row, or at least make it configurable. Bold text is not enough to identify the heading row.

Please add an external border to the table. Not having the external border exacerbates the issue of not having enough space above the table.

Here's a screen shot that shows all of these issues.

![image](https://github.com/GitbookIO/community/assets/107077736/01cb4569-7778-448a-b741-e3bd33f551f1)

**Font mismatch in hints text**

While most hints are paragraphs, in a few cases I do need to include lists. When I do that, the paragraph font and the list font are not the same. The list font is just the regular display font, while the hint font is slightly smaller.

![HintDisplayIssue](https://github.com/GitbookIO/community/assets/107077736/e7a1676a-be9c-44d0-bc4b-6ca20e31a9e9)

**Comment by JaniceManwiller** [comment]
Also noticed an issue with direct links to anchors within another topic. Previously, the link text was the heading of the destination section.

It's now changed to `Topic title#anchor-name`. The anchor name isn't really intended to be visible to users - I always adjust them to use a semi-standard format that is not identical to the title.

Here is an example of a direct link to a Topic subsection.

![LinkToAnchor](https://github.com/GitbookIO/community/assets/107077736/af2c7fef-0bbf-4710-9aaf-b071c053bbd3)

The destination is this subtopic.

![LinkDestination](https://github.com/GitbookIO/community/assets/107077736/73b4a072-5a33-4c0c-af57-719302d9f98f)

**Reply by ahcsco** [reply]
Agree with alignment. I think it would be also beneficial to have the possibility to merge and split cells in the table.

**Comment by SamyPesse** [comment]
> I've been saying this for a long time, and reported it before, but table cell content really needs to be vertically aligned to the top of the cell, not the middle of the cell. I thought when I previewed the new design it was vertically aligned to the top, but now that it's out it's center aligned again. It makes the table content very difficult to read.

While I agree, unfortunately, we had to revert this change as it hurt many other users who relied on the previous alignment. We'll explore a solution to support top alignment in the future.


> Also noticed an issue with direct links to anchors within another topic. Previously, the link text was the heading of the destination section.
> It's now changed to Topic title#anchor-name. The anchor name isn't really intended to be visible to users - I always adjust them to use a semi-standard format that is not identical to the title.

This one was fixed yesterday and should now work as expected.

> There is not enough space above the tables.

Can you share the url of the content with the table?

> Please restore the shading to the heading row, or at least make it configurable. Bold text is not enough to identify the heading row.

We'll look into it.

> While most hints are paragraphs, in a few cases I do need to include lists. When I do that, the paragraph font and the list font are not the same. The list font is just the regular display font, while the hint font is slightly smaller.

We'll look into this one as well.

**Comment by JaniceManwiller** [comment]
So the same issues (verticial cell alignment, heading row format, space above) are present for all of my tables, the URL for the table I took the image of is at https://docs.tonic.ai/app/generation/generators/generator-summary

A few more examples:

https://docs.tonic.ai/app/readme-1/tonic-license-plans#tonic-license-plans-feature-comparison
https://docs.tonic.ai/app/admin/tonic-user-access/permissions/available-permissions
https://docs.tonic.ai/app/setting-up-your-database/data-connector-summary
